WalletWasabi.Fluent/ViewModels/Wallets/Settings/ExcludedCoinsViewModel.cs

```diff
@@ -0,0 +1,75 @@
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Windows.Input;
+using DynamicData;
+using DynamicData.Aggregation;
+using DynamicData.Binding;
+using ReactiveUI;
+using WalletWasabi.Fluent.Infrastructure;
+using WalletWasabi.Fluent.Models.Wallets;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Fluent.ViewModels.Wallets.Coins;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Settings;
+
+[AppLifetime]
+[NavigationMetaData(Title = "Excluded Coins", NavigationTarget = NavigationTarget.DialogScreen)]
+public partial class ExcludedCoinsViewModel : DialogViewModelBase<Unit>
+{
+	private readonly IWalletModel _wallet;
+	private readonly CompositeDisposable _disposables = new();
```

Comment:
Unnecessary, use the one that is provided in the OnNavigatedTo()

Reply:
Fixed